timeless/reservations/views.py

```diff
@@ -44,6 +46,31 @@ class CommentView(CrudAPIView):
     url_lookup = "comment_id"
 
 
+class JsonView(views.MethodView):
+    """ Reservation JSON API """
+
+    def post(self):
+        """Retrieve reservations based on login, location, and date.
+            @todo #28:30min Implement actual fetching of reservations. We need to
+             return a JSON list, filtered by reservations based on location and
+             date. We also need to password protect this API, and filter only
+             those belonging to the specific company,
+        :param company_id:
+        :return:
+        """
+        reservations_json = {
```

Comment:
@carlosmiranda @emilianodellacasa 
I think we should return puzzle for implementation serialization and deserialization objects using this lib - https://flask-marshmallow.readthedocs.io/en/latest/ if we provide API.
if @emilianodellacasa agree with it, @carlosmiranda pls leave a puzzle here

Reply:
@vladarefiev I think we should